### ISSUE TITLE
Allow shift+click to select multiple items in Storage bucket

### ISSUE THREAD

**Issue by saltcod** [supabase/supabase#11317]
Would be great to be able to select a group of items in a range via shift+click in a Storage bucket 
<img width="58" alt="CleanShot 2022-12-28 at 06 42 38@2x" src="https://user-images.githubusercontent.com/105593/209796211-2e3a52b5-4da1-4756-aa86-3a4e10fb6cf1.png">

**Comment by itisnajim** [comment]
it will be great too in the database table editor

**Comment by Calebm5577** [comment]
Hello, is anyone working on this? I would like to if no one else already is. I cloned the repo and followed the getting started guide, only issue is under projects in app.supabase (http://localhost:8082/), clicking on buckets or anything for that matter (table editor, database, etc) results in an infinite load animation. I replicated the issue on the live website but unable to get this to work locally thus far.
